src/migrations/capybara.js

```diff
@@ -1,6 +1,6 @@
 import path from 'path';
-import SDKMigration from './base';
-import { codeshift, run } from '../utils';
+import SDKMigration from './base.js';
+import { codeshift, run, ROOT } from '../utils.js';
```

Comment:
Gonna nitpick the same const-first preference here. Just pretend I wrote this comment for all the migration files.